### DISCUSSION TITLE
Is it possible to left/centre/right align the map box?

### DISCUSSION THREAD

**Discussion by MattMGK** [javalent/obsidian-leaflet#403] [Q&A]
Pretty much as the title says. The map box (I apologise I don't know the proper word for it, if there is one) seems to automatically take up the centre of the note with smaller dimensions, wondering if it's possible to left/right align it? Or even if I could choose which side it expands into when increasing the width. 

So ideally in the below image it would fill up all of the blank space to the left of it. As in the image below. However when I increase the width it expands in both directions, causing it to clash with the wikibox on the right side dropping it below it, like in the second image.
![image](https://github.com/javalent/obsidian-leaflet/assets/65480746/975630cc-2b54-4b68-8cc3-a2794a5f8424)
![image](https://github.com/javalent/obsidian-leaflet/assets/65480746/3dc58b99-4ebc-41bb-9c6b-6eeee06d1345)

In case it matters, the Wikibox is coming from ITS These/  Style Settings.

Thanks in advance for any help y'all can provide!!!

**Comment by MattMGK** [comment]
So not an actual solution, but a work around. So if it's not a thing you can do natively in the plugin yet, would be great if it was added!

Workaround - Simply add > before each line of Leaflet code. Like in the image below.
![Obsidian_6dfa0dLqOt 2023-07-02 at 01-28-13-AM](https://github.com/javalent/obsidian-leaflet/assets/65480746/3bd3f403-bfea-4f48-931d-ca119217890a)

**Comment by EthanDunning** [comment]
Late, but I put the leflet code in its own note, and then where ever I needed to put the map, I created a table with 1 column and 1 row and put a reference in there like below:
| ![[Map]] |
| ------------------- |
